nodeclient/indexer_models.go

```diff
@@ -74,6 +74,12 @@ type IndexerNativeTokenParas struct {
 	MaxNativeTokenCount *uint32 `qs:"maxNativeTokenCount,omitempty"`
 }
 
+// IndexerUnlockableByAddressParas define address unlock related query parameters.
+type IndexerUnlockableByAddressParas struct {
+	// Bech32-encoded address that should be searched for.
+	UnlockableByAddressBech32 string `qs:"unlockableByAddress,omitempty"`
```

Comment:
why is it omitempty if that is the only criteria?

Reply:
copy-paste

Reply:
although, it needs to be omitempty, because there is always a default value in go